### ISSUE TITLE
TeslaSwiftDemo, client ID error message

### ISSUE THREAD

**Issue by Jim3750** [jonasman/TeslaSwift#169]
I am trying to run teslaSwiftDemo. 

Not sure this is correct but I had to change the "Team" info in "Signing and Capabilities" to the one I use for my own app (Pilot Log for Tesla), to get rid of the message "No account for team RMBS8HAUHZ".

When I run the app, I get the error message "We do not recognize this client ID".

Sorry matter is probably basic, but help is appreciated.

**Comment by jonasman** [comment]
You need to use your client ID, check the readme
Sent from my iPhone

On 7 Apr 2024, at 15:25, Jim3750 ***@***.***> wrote:

﻿

I am trying to run teslaSwiftDemo.

Not sure this is correct but I had to change the "Team" info in "Signing and Capabilities" to the one I use for my own app (Pilot Log for Tesla), to get rid of the message "No account for team RMBS8HAUHZ".

When I run the app, I get the error message "We do not recognize this client ID".

Sorry matter is probably basic, but help is appreciated.

**Comment by Jim3750** [comment]
Thank you for the fast answer.

In order to run TeslaSwiftDemo, can I use the clientID / clientSecret / redirectURI obtained registering my own app or do I need to register the TeslaSwiftDemo app under my Tesla account?

In case I need to register TeslaSwiftDemo, what is the « Allowed Origin URL »?

Thank you.

**Comment by Jim3750** [comment]
I have finally managed to (start to) run TeslaSwiftDemo. I went thru the Tesla login and I ended with the following error: "Account must be registered in the current region https://fleet-api.prd.eu.vn.cloud.tesla.com".
I fulfilled today the register step (using domain where public key is stored, CSR, partner auth token). 
Next step indicated in Tesla doc is "Wait for Tesla to process your CSR. This may take up to two weeks. Once complete, you will receive an email from Tesla."
I suppose I need to wait for this before re-testing the app.
Looks OK to you?
Thks.
